### PR TITLE
feat(scss): refactor styles

### PR DESCRIPTION
**Pull request by KyleMcMaster**
- move away from bootswatch bs4 stylesheet to bootstrap classes directly
- implement custom styles as needed 

## Purpose

- [x] Feature
- [ ] Partial Feature
- [ ] Bugfix
- [ ] Docs, Config
- [ ] Chore / Format

## Change Type

- [x] Major
- [ ] Minor
- [ ] Fix

## Data Persistence Changes

n/a

## Configuration Changes

n/a

### CODE REVIEW

File: client/src/styles.scss [hunk 1]
```diff
@@ -1,60 +1,80 @@
...
+  background-color: $base-color;
+}
+
+.full-height {
+  height: calc(100vh - 60px);
+}
+
+.background-dark {
+  background-color: #000;
+  background-color: var($base-color);
```
Comment (eyev) [inline client/src/styles.scss:50]: this should contain a css variable name in it... `$base-color` is a variable, but it belongs to sass and will never find its way into the browser (whereas, css vars will)

Comment (KyleMcMaster) [reply client/src/styles.scss]: i believe this is fixed

File: client/src/styles.scss [hunk 2]
```diff
@@ -1,60 +1,59 @@
 /* You can add global styles to this file, and also import other style files */
 // Required for bs4
-@import '~bootstrap/scss/functions';
-@import '~bootstrap/scss/variables';
-@import '~bootstrap/scss/mixins';
+@import '~bootstrap/scss/bootstrap';
```
Comment (KyleMcMaster) [inline client/src/styles.scss:3]: this might no longer be needed?

Comment (KyleMcMaster) [reply client/src/styles.scss:3]: still needed

**Comment by KyleMcMaster** [comment]
LGTM

**Review by seangwright — APPROVED** [review]

**Pull request merged**